fty-nutconfig

```diff
@@ -72,10 +74,35 @@ EOF
 RES=$?
     if [ "$RES" = 0 ] ; then
         chown -R root:root /etc/systemd/system/nut-driver@.service.d/
-        /bin/systemctl daemon-reload
+        SYSTEMCTL_RELOAD=true
     fi
 fi
 
+if [ "$RES" = 0 ] \
+&& [ ! -s /etc/systemd/system/nut-driver-enumerator-daemon.service.d/loopfreq.conf ] \
+&& /bin/systemctl is-enabled nut-driver-enumerator-daemon.service | grep -w enabled \
+; then
+    mkdir -p /etc/systemd/system/nut-driver-enumerator-daemon.service.d/
+    cat << EOF > /etc/systemd/system/nut-driver-enumerator-daemon.service.d/loopfreq.conf
```

Comment:
@geraldguillaume @kkathreen : the resync frequency in `nut-driver-enumerator-daemon` can have performance implications, so I made it easier to tune it when you would experiment to scale the system.

While there should not be much about systems activity when there are no changes to apply, there is still a cost of coming to that decision, proportional to amounts of `ups.conf` sections and `nut-driver@*.service` instances and thus a bigger or smaller portion of system time dedicated to this. There is also quite a bit of optimization that can be achieved in NDE script itself, if needed; but to tinker with that it would be useful to know if the current approach is good enough or becomes a bottleneck at some point.

My expectation is that with other development in https://github.com/42ity/nut/pull/70#issuecomment-444612942 the daemon should reliably process even the rapid changes to `ups.conf`; if that proves true in the field, this loop delay can become longer.